src/promotion-builder/common/navigation/links/Links.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import Links from './Links'
+
+describe('<Links>', () => {
+
+    let wrapper;
+    let clickHandlerMock;
+
+    beforeEach(() => {
+        clickHandlerMock = jest.fn();
+        wrapper = shallow(<Links className={'devices'} clickHandler={clickHandlerMock} href={'hrefValue'} text={'mobile'}/>);
+    })
+
+    it('renders with type', () => {
+        expect(wrapper.type()).toBe('li');
+    })
+
```

Comment:
No need for the describe. Maybe an `it` block which says it renders the correct props, then add all the assertions in there